### ISSUE TITLE
Virtual Display Driver now automatically disables all other montiors, even when Sunshine is not in use

### ISSUE THREAD

**Issue by sam-ro159** [Cynary/sunshine-virtual-monitor#18]
Hello. I had been spending some time trying to set this up, and I was successful in getting it to work once. However, when I returned to my computer, both monitors were still disabled. Not a big deal, as I can simply use the pnputil script in the Readme. The monitors where re-enabled and everything was fine.

However, now, even when Sunshine is not in use, every time I enable the Virtual Display Driver, my monitors become disabled. I've re-installed the iddSampleDriver multiple times, but even in the installation process it will disable my other monitors.

The virtual display was working prior to using the scripts. 

Is it something within one of the scripts that I can disable, so I can try and use the virtual display again, or would it be something within the virtual display driver? Or would it be in the MultiMonitorTool or WindowsDisplayManager? Either way, I'm not sure where to go from here.

Thank you.

**Comment by LeGeRyChEeSe** [comment]
Hello, this is due to the preferences kept in memory by Windows on the location of the screens depending on which screens are connected at the same time or not.
e.g.:
- Monitor 1 connected + Monitor 2 **connected** = Configuration n°1
- Monitor 1 connected + Monitor 2 **disconnected** = Configuration n°2
- etc.

So you just have to manually activate the physical screens after activating the virtual screen to keep this configuration in memory and the problem will be solved.

**Comment by Cynary** [comment]
Yeah, this is an annoying issue - that's what the teardown script tries to avoid by fully disabling the virtual display when you're not connected.
If this happens, you can recover even without seeing a picture, the README has the disclaimer section with some tips on how to do it: https://github.com/Cynary/sunshine-virtual-monitor?tab=readme-ov-file#disclaimer
